lib/metadataTypes/Campaign.js

```diff
@@ -113,6 +51,26 @@ class Campaign extends MetadataType {
         }
         return metadataStructure;
     }
+
+    /**
+     * Parses campaign asset response body and returns metadata entries mapped to their id
+     * @param {String} retrieveDir folder where to save
+     * @param {String} id of camapaign to retrieve
+     * @param {String} name of camapaign for saving
+     * @returns {Promise<Object>} Campaign Asset Object
+     */
+    static async getAssetTags(retrieveDir, id, name) {
+        const res = await this.client.rest.getBulk(`/hub/v1/campaigns/${id}/assets`);
+
+        for (const asset of res.items) {
+            File.writeJSONToFile(
+                retrieveDir + '/campaign/' + name + '/assets/',
```

Comment:
inconsistent quote vs templating. please use one or the other (compare lines 63 and 67)